ml/rlvr/training_collector.py

```
@@ -0,0 +1,374 @@
+                    "prompt": example['prompt'],
+                    "completion": example['completion'],
+                    "reward_score": Decimal(str(example['reward_score'])),
+                    "example_type": example['example_type'],
+                    "is_synthetic": False,
+                    "is_validated": True,  # Already verified
+                    "created_at": datetime.utcnow()
+                })
+
+                saved_count += 1
+
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)